x-pack/plugins/canvas/public/components/asset_manager/asset_manager.js

```diff
@@ -81,31 +87,44 @@ export class AssetManager extends React.PureComponent {
         <EuiSpacer size="s" />
 
         <EuiFlexGroup alignItems="baseline" justifyContent="center" responsive={false}>
+          <EuiFlexItem className="asset-create-image" grow={false}>
+            <EuiToolTip content="Create image element">
+              <EuiButtonIcon
+                iconType="plusInCircle"
```

Comment:
We use `vector` for the "Add element" control, I think it would make sense, and probably be more consistent, to do the same here.